### PR TITLE
fix: November dependency bumps 🌲

### PR DESCRIPTION
**Pull request by stdavis**
Also include some lint fixes and styling

### CODE REVIEW

File: packages/dart-board/package.json [hunk 1]
```diff
@@ -53,7 +53,7 @@
   "dependencies": {
     "@ugrc/utilities": "^2.2.1",
     "clsx": "^1.2.1",
-    "reactstrap": "^9.1.4"
+    "reactstrap": "^9.1.5"
```
Comment (steveoh) [inline packages/dart-board/package.json:56]: **thought**: i always wonder if we should be loose on these widget dependencies. Like `> 2.0.0` etc.

File: packages/sherlock/src/Sherlock-InMap.stories.jsx [hunk 2]
```diff
@@ -3,9 +3,10 @@ import Sherlock, { MapServiceProvider } from './';
 import Map from '@arcgis/core/Map';
 import MapView from '@arcgis/core/views/MapView';
 import '@arcgis/core/assets/esri/themes/light/main.css';
-import * as watchUtils from '@arcgis/core/core/watchUtils';
+import * as reactiveUtils from '@arcgis/core/core/reactiveUtils';
```
Comment (steveoh) [inline packages/sherlock/src/Sherlock-InMap.stories.jsx:6]: **praise**: nice!

**Review by steveoh — APPROVED** [review]

File: packages/dart-board/package.json (hunk 1 above)
Comment (stdavis) [reply packages/dart-board/package.json:56]: I didn't know that you could specify npm deps like that. Hmmm

Comment (steveoh) [reply packages/dart-board/package.json:56]: don't we do it with react that way?

**Pull request merged**